extension/src/popup/components/account/AccountOptionsDropdown/styles.scss

```diff
@@ -0,0 +1,37 @@
+@use "../../../styles/utils.scss" as *;
+
+.AccountOptionsDropdown {
+  &__modal {
+    background: var(--sds-clr-gray-01);
+    border-radius: #{pxToRem(6px)};
```

Comment:
I'm trying to make a conscious effort to start using this pxToRem helper more. It will probably make our css a little more foolproof if we let the helper do the math for us